fluent-bundle/examples/typesafe_messages.rs

```diff
@@ -0,0 +1,129 @@
+use std::borrow::Cow;
```

Comment:
Suggestion: Please add a description here at the top of what this is demonstrating, so that users can quickly gain context.